exportcontrol/urls.py

```diff
@@ -15,6 +15,7 @@
     path("accounts/", include("django.contrib.auth.urls")),
     path("accounts/", include("accounts.urls")),
     path("", search_page, name="search_page"),
+    path("regulation/<str:regulation_code>", regulation_page, name="regulation_page"),
```

Comment:
the url must be inside the application and after imported, but I can do it later too